service/src/test/java/org/databiosphere/workspacedataservice/service/ImportServiceTest.java

```diff
@@ -55,7 +57,7 @@
 
 @ActiveProfiles(profiles = {"mock-sam", "mock-collection-dao"})
```

Comment:
```suggestion
@ActiveProfiles(profiles = {"mock-sam", "mock-collection-dao", "noop-scheduler-dao"})
```

If you do this, you won't have to stub out scheduler throughout.